lib/stream/consumers.js

```diff
@@ -50,6 +51,14 @@ async function buffer(stream) {
   return Buffer.from(await arrayBuffer(stream));
 }
 
+/**
+ * @param {AsyncIterable|ReadableStream|Readable} stream
+ * @returns {Promise<Uint8Array>}
+ */
+async function bytes(stream) {
+  return new Uint8Array(await buffer(stream));
```

Comment:
```suggestion
  return new Uint8Array(await arrayBuffer(stream));
```

There's no need to create an intermediate `Buffer` just to then copy its contents into the new `Uint8Array`.